trl/trainer/sft_trainer.py

```diff
@@ -424,12 +423,6 @@ def make_inputs_require_grad(module, input, output):
         elif self.args.max_steps == -1 and args.packing:
             self.train_dataset.infinite = False
 
-        if any(isinstance(callback, RichProgressCallback) for callback in self.callback_handler.callbacks):
```

Comment:
For some reason this was only included in the SFT trainer and no others - I removed it for consistency